api/types/crds/skupper_site_crd.yaml

```diff
@@ -20,14 +20,9 @@ spec:
                 linkAccess:
                   type: string
                 settings:
-                  type: array
-                  items:
-                    type: object
-                    properties:
-                      name:
-                        type: string
-                      value:
-                        type: string
+                  type: object
+                  additionalProperties:
+                    type: string
```

Comment:
I have needed to change slightly the CRD Because the controller is expecting a map of strings for the site settings, instead of an array of objects with `name` and `value`. Also, With this modification, `kubectl describe` returns something like this:
```
Spec:
  Link Access:  default
  Settings:
    Ingress - Host:  0.0.0.0
    Name:            example
```
On the other hand, with an array of objects it would return something like this:
```
Spec:
  Link Access:  default
  Settings:
    Name:   Ingress - Host
    Value:  0.0.0.0
    Name:   Name
    Value:  example
```